docs/sources/project/building_from_source.md

```diff
@@ -3,7 +3,7 @@ title = "Building from source"
 type = "docs"
 [menu.docs]
 parent = "installation"
-weight = 5
+weight = 700
```

Comment:
Why is this topic in the Projects folder rather than Installation?

Reply:
because moving it means a URL change and need to add redirect, and we had a plan to restructure all the files and wanted to wait for that and do everything at once to avoid too many 301 redirects (SEO is important for the docs)